io-native/src/main/java/org/ballerinalang/stdlib/io/nativeimpl/CharacterChannelUtils.java

```diff
@@ -129,11 +129,11 @@ public static Object readString(BObject channel) {
             for (int i = 0; i < lines.length; i++) {
                 joiner.add(lines[i]);
             }
-            return io.ballerina.runtime.api.utils.StringUtils.fromString(joiner.toString());
+            return io.ballerina.runtime.api.utils.StringUtils.fromString(joiner.toString()
+                    .replaceAll(System.lineSeparator(), "\n"));
```

Comment:
Why do we have fully-qualified class name for `StringUtils` here. We could just use `StringUtils.fromString(...)` right ?

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-io/pull/200/commits/42c572932d4f44f19e59aa51b172c6d56e0f3572